erpnext/manufacturing/doctype/work_order/test_work_order.py

```diff
@@ -371,6 +371,49 @@ def test_work_order_with_non_transfer_item(self):
 		ste1 = frappe.get_doc(make_stock_entry(wo.name, "Manufacture", 1))
 		self.assertEqual(len(ste1.items), 3)
 
+	def test_work_order_with_batch_size(self):
```

Comment:
lets rename this to `test_job_card_time_with_batch_size` ? The test should describe what we're testing.